### ISSUE TITLE
[Feature] Reminders

### ISSUE THREAD

**Issue by olets** [olets/zsh-abbr#147]
Support "reminder" messages if you missed an opportunity to use an abbreviation.

```shell
% abbr x=a
% a[ENTER]
Could have used the abbreviation `x`
```

Will check the entire buffer (matches `^.+$`-like), and then will start trimming words off the front of the buffer (matches `.+$`-like)

```shell
% abbr x=a
% b a[ENTER]
Could have used the abbreviation `x`
% a b[ENTER]
# no message
```

**Comment by olets** [comment]
This is in v6.0.0-beta.1

- Bug: #154

**Comment by olets** [comment]
Released in v6.0.0
